sysutils/vbindiff/Portfile

```diff
@@ -14,13 +13,14 @@ long_description    VBinDiff (Visual Binary Diff) displays files in \
                     them. Unlike diff, it works well with large files (up to \
                     4 GB).
 
-homepage        http://www.cjmweb.net/vbindiff/
+homepage        https://www.cjmweb.net/vbindiff/
 platforms       darwin
 license         GPL-2+
 master_sites    ${homepage}
 
-checksums       md5 dbda80ef580e1a0975ef50b9aaa5210e \
-                sha1 0e8c63debafe3d5bb8741e0254c95457d78c7f2d \
-                rmd160 156fa68891d4aa9c0280177c70e1a3d53b912d13
+checksums       md5     bbde86b790d6a912e3ba11d6cd8b1a2a \
+                sha1    cbd4651f604ad18cf8d9122b3c2719d0173218f7 \
```

Comment:
Please remove the "md5" and "sha1" checksums, we no longer use those. You should also add a "size" field at the end while you're at it.